paris-processor/src/main/java/com/airbnb/paris/processor/writers/StyleExtensionsKotlinFile.kt

```diff
@@ -1,10 +1,9 @@
 package com.airbnb.paris.processor.writers
 
-import com.airbnb.paris.processor.EXTENSIONS_FILE_NAME_FORMAT
-import com.airbnb.paris.processor.PARIS_KOTLIN_EXTENSIONS_PACKAGE_NAME
-import com.airbnb.paris.processor.STYLE_CLASS_NAME
+import com.airbnb.paris.processor.*
```

Comment:
should the kotlin style guide be set as the style guide for the project and saved in version control? my formatter was trying to do different things and it would be nice to make the rules consistent

Reply:
Yes I'm planning on doing this. I'll probably copy over whatever Epoxy has set up.